.ark.env.local

```diff
@@ -4,4 +4,5 @@ ARK_QUICKSTART_BASE_URL=https://azure.prod.com
 ARK_QUICKSTART_API_VERSION=2024-04-01-preview
 ARK_QUICKSTART_API_KEY=
 ARK_QUICKSTART_PROMPT_YES=yes
-ARK_QUICKSTART_CONTROLLER_IMAGE=
\ No newline at end of file
+ARK_QUICKSTART_CONTROLLER_IMAGE=
+ARK_DETECT_NAMESPACE=
```

Comment:
I think we can remove this env var completely - will discuss later in the PR to show how